jconstraints-runner/src/main/java/statistics/StrInReAssertionCounter.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2015 United States Government, as represented by the Administrator
+ *                of the National Aeronautics and Space Administration. All Rights Reserved.
+ *           2017-2021 The jConstraints Authors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package statistics;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.PathMatcher;
+import java.nio.file.Paths;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import statistics.callables.StrInReAnalysisTask;
+import statistics.datastrucutres.BooleanAnswer;
+
+public class StrInReAssertionCounter {
+
+  ExecutorService pool;
+  List<Future<BooleanAnswer>> tasks;
+
+  public StrInReAssertionCounter() {
+    int usedCores = Runtime.getRuntime().availableProcessors() - 2;
+    pool = Executors.newWorkStealingPool(Runtime.getRuntime().availableProcessors() - 2);
+    tasks = new LinkedList<>();
+    System.out.println(String.format("Created Work-Stealing Pool with %d cores", usedCores));
+  }
+
+  public static void main(String args[]) {
```

Comment:
IDE warning: C-Style Array (valid, but uncommon in Java)
```suggestion
  public static void main(String[] args) {
```